lib/smart_proxy_dns_powerdns/dependencies.rb

```diff
@@ -8,6 +8,9 @@ class Proxy::Dns::DependencyInjection::Dependencies
   when 'postgresql'
     require 'smart_proxy_dns_powerdns/backend/postgresql'
     dependency :dns_provider, Proxy::Dns::Powerdns::Backend::Postgresql
+  when 'rest'
```

Comment:
I intend to do that, but I think I'm going to finish this first and then do a release. That should make it possible to run it on 1.12. After that I'll move to 1.13+.